Emby.Server.Implementations/Data/SqliteUserRepository.cs

```diff
@@ -73,6 +74,37 @@ private void TryMigrateToLocalUsersTable(ManagedConnection connection)
             }
         }
 
+        private void RemoveEmptyPasswordHashes()
+        {
+            foreach (var user in RetrieveAllUsers())
+            {
+                // If the user password is the sha1 hash of the empty string, remove it
+                if (!string.Equals(user.Password, "DA39A3EE5E6B4B0D3255BFEF95601890AFD80709") || !string.Equals(user.Password, "$SHA1$DA39A3EE5E6B4B0D3255BFEF95601890AFD80709"))

```

Comment:
```suggestion
                if (!string.Equals(user.Password, "DA39A3EE5E6B4B0D3255BFEF95601890AFD80709", StringComparison.Ordinal)
                    || !string.Equals(user.Password, "$SHA1$DA39A3EE5E6B4B0D3255BFEF95601890AFD80709", StringComparison.Ordinal))
```
Depending if they need to be case sensitive

Reply:
I would rather we don't hardcode sha1 hashes of empty string but compute them, probably as a class-level static.

Reply:
It's a migration. It's a one-off.